Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist

```diff
@@ -607,6 +845,6 @@ A profile can consist of payloads with different version numbers. For example, c
 	<key>pfm_unique</key>
 	<false/>
 	<key>pfm_version</key>
-	<integer>3</integer>
+	<integer>5084</integer>
```

Comment:
`pfm_version` refers to the version of this file. So this update is revision `4`.

Reply:
Fixed pfm_version